### ISSUE TITLE
Schemas Validation

### ISSUE THREAD

**Issue by FrancisBases** [SchildCode/EpXL#9]
After downloading EpXL_v190926 > 
Then CTRL_I to import file https://drive.google.com/file/d/1NnADJgAVzK--277nchesAwjYxK9ILlDc/view?usp=sharing (basically a subset of **CmplxGlz_MeasuredDeflectionAndShading.idf**

All is imported without errors.

If clicking in the Schema > Yes to update to Version 9.1 (from my install) > CTRL_I to import the same file then it returns the following:
![image](https://user-images.githubusercontent.com/24868420/65714289-b51e0000-e092-11e9-8ca2-3292cb49bd97.png)

It could be because it is using 0.04 and not 0,04.

Cheers.

**Comment by SchildCode** [comment]
Fixed in v190927.
- The Schema is now stored in the correct way irrespective of regional settings (comma or point). I hadn’t though about the fact that also users with comma-decimal need to import the schema.
- Incidentally, I also improved the code in CTRL_Q, so that it properly differentiates between inclusive and exclusive min/max bounds, also for extented fields and mixed-type fields (number or string)
